src/main/java/net/sf/jabref/preferences/JabRefPreferences.java

```diff
@@ -303,7 +303,6 @@
     public static final String SHOW_FILE_LINKS_UPGRADE_WARNING = "showFileLinksUpgradeWarning";
     public static final String SIDE_PANE_WIDTH = "sidePaneWidth";
     public static final String LAST_USED_EXPORT = "lastUsedExport";
-    public static final String LAST_USED_IMPORT = "lastUsedImport";
```

Comment:
Yeah, you are right, forgot that new feature. Then it's ok to be removed.

2016-08-19 11:58 GMT+02:00 Stefan Kolb <notifications@github.com>:

> In src/main/java/net/sf/jabref/preferences/JabRefPreferences.java
> <https://github.com/JabRef/jabref/pull/1774#discussion_r75456162>:
>
> > @@ -303,7 +303,6 @@
> >      public static final String SHOW_FILE_LINKS_UPGRADE_WARNING = "showFileLinksUpgradeWarning";
> >      public static final String SIDE_PANE_WIDTH = "sidePaneWidth";
> >      public static final String LAST_USED_EXPORT = "lastUsedExport";
> > -    public static final String LAST_USED_IMPORT = "lastUsedImport";
>
> Because all supported filters are now default so you can definitely see
> possible files. If the importer has problems with auto recognition one can
> select the import format by hand (although this should not happen). Just
> wanted to remove complexity here. Maybe it's still needed - we have to
> discuss this...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/JabRef/jabref/pull/1774/files/a3656a640c1ebb6b43447152a25fbed7b90a1a66#r75456162>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATi5COhrZNkN301wn-NogiV6caWxPP5ks5qhX44gaJpZM4Jnghh>
> .
>